handler/EventHandler.go

```diff
@@ -0,0 +1,209 @@
+package handler
```

Comment:
File names should be lower cased and snake cased where necessary - event_handler rather than EventHandler